src/interpreter/contract.rs

```diff
@@ -44,6 +45,98 @@ where
         self.inc_pc()
     }
 
+    pub(crate) fn transfer(&mut self, a: Word, b: Word, c: Word) -> Result<(), RuntimeError> {
+        let (ax, overflow) = a.overflowing_add(32);
+        let (cx, of) = c.overflowing_add(32);
```

Comment:
Instead of 32 here, can we use something like `ContractId::LEN` and/or `Color::LEN`?